packages/dht/src/connection/websocket/WebsocketServerConnector.ts

```diff
@@ -114,23 +114,27 @@ export class WebsocketServerConnector {
     private attachHandshaker(connection: IConnection) {
         // TODO: use createIncomingHandshaker here?
         const handshaker = new Handshaker(this.localPeerDescriptor!, connection)
-        handshaker.once('handshakeRequest', (localPeerDescriptor: PeerDescriptor, sourceVersion: string, remotePeerDescriptor?: PeerDescriptor) => {
-            this.onServerSocketHandshakeRequest(localPeerDescriptor, connection, handshaker, sourceVersion, remotePeerDescriptor)
+        handshaker.once('handshakeRequest', (
+            localPeerDescriptor: PeerDescriptor,
+            remoteProtocolVersion: string,
```

Comment:
The parameter name was `source...` before this PR, now `remote...`. Is this an intentional change?

(If it is, should we rename `source`->`remote` in some other places, too? In a separate PR maybe.)

Reply:
Source can be used in cases where there is no clear interaction between two parties. Examples would be routing and recursive operations

Reply:
Are there some usage of `source` naming convention where the interaction is clear and therefore `local`/`remote` would be better?  (e.g. `WebsocketServerConnector`?)

Reply:
https://linear.app/streamr/issue/NET-1392/unify-naming-between-remotelocal-and-source-patterns

Reply:
Yes in the connectors it makes sense to use remote / local, as all individual interactions are between two parties